tests/util/mod.rs

```diff
@@ -258,3 +258,18 @@ macro_rules! expect_read {
         assert_eq!(address, source);
     }};
 }
+
+/// Call all registration operations, ending with `source` being registered with `token` and `final_interests`.
+pub fn registry_ops_flow(
```

Comment:
Since its only used in `tests/poll.rs` it can live there.

Reply:
ok, moved.